circle.yml

```diff
@@ -0,0 +1,53 @@
+machine:
+  python:
+    version: 2.7.3
+
+general:
+  artifacts:
+    - "reports"
+    - "test_root/log"
+
+dependencies:
+  override:
+    - npm install
+    - bundle install
+
+    - pip install setuptools==0.6c11
+    - pip install distribute==0.6.49
+    - pip install -r requirements/edx/paver.txt
+
+    # Mirror what paver install_prereqs does.
+    # After a successful build, CircleCI will
+    # cache the virtualenv at that state, so that
+    # the next build will not need to install them
+    # from scratch again.
+    - pip install -r requirements/edx/pre.txt
+    - pip install -r requirements/edx/github.txt
+    - pip install -r requirements/edx/local.txt
+
+    # HACK: within base.txt stevedore had a
+    # dependency on a version range of pbr.
+    # Install a version which falls within that range.
+    - pip install pbr==0.9.0
+    - pip install -r requirements/edx/base.txt
+    - pip install -r requirements/edx/post.txt
+
+    - pip install coveralls
+
+test:
+  override:
+    - ./scripts/all-tests.sh:
+        parallel: true
```

Comment:
Maybe a comment about what the parallel setting does (even if it's already in the circle docs)?